.gitignore

```diff
@@ -9,8 +9,6 @@
 *.fls
 *.log
 *.synctex.gz
-# Python virtual environment
-/.venv/
```

Comment:
It's ok to have this in the `.gitignore` if it's useful for you. Nobody else will have a problem with this.

Reply:
No, it's not useful anymore. I realised it's more efficient to use one env for all blueprint-driven projects.